box-content-sdk/src/main/java/com/box/androidsdk/content/models/BoxMetadataUpdateTask.java

```diff
@@ -0,0 +1,83 @@
+package com.box.androidsdk.content.models;
+
+/**
+ * BoxMetadataUpdateTask instances are used for updating metadata with BoxMetadataUpdateRequest.
+ */
+public class BoxMetadataUpdateTask extends BoxJsonObject {
```

Comment:
I think it might be cleaner if we don't expose another object for developers to have to worry about. Can we expose a method on the ```BoxRequestsMetadata.Update#addTask(String op, String path, String value)``` and construct this object within the update request object?